### PR TITLE
send list name in inViewerList.js

### PR DESCRIPTION
**Pull request by sumitks866**

### CODE REVIEW

**Comment by Aakash-Jakhmola** [comment]
@sumitks866  add this to searchMovie as well

**Comment by sumitks866** [comment]
> @sumitks866 add this to searchMovie as well

done

File: src/domain/movie/getMovieList.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const { User, Watch } = require("../../database/models");
 const ErrorHandler = require("../../utils/errorHandler");
-const hasViewerWatched = require("./hasViewerWatched");
+const hasViewerWatched = require("./inViewerList");
```
Comment (Aakash-Jakhmola) [inline src/domain/movie/getMovieList.js]: rename `hasViewerWatched` to `inViewerList`

File: src/domain/movie/getTrending.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const Tmdb = require('./../../utils/tmdb');
-const hasViewerWatched = require('./hasViewerWatched');
+const hasViewerWatched = require('./inViewerList');
```
Comment (Aakash-Jakhmola) [inline src/domain/movie/getTrending.js]: here  also

Comment (Aakash-Jakhmola) [reply src/domain/movie/getTrending.js]: @sumitks866  please fix it

Comment (sumitks866) [reply src/domain/movie/getTrending.js]: done. check it now
if any minor changes still remains ... diy

**Pull request merged**